src/store/login/login.thunks.ts

```diff
@@ -236,11 +246,20 @@ function postCreateAccountAuthorization(wallet: HermezWallet.HermezWallet): AppT
         dispatch(loginActions.addAccountAuthSuccess());
         dispatch(push(redirectRoute));
       } catch (error: unknown) {
-        console.error(error);
-        const text = adapters.parseError(error);
-        dispatch(loginActions.addAccountAuthFailure(text));
-        dispatch(globalActions.openSnackbar({ type: "info-msg", text }));
-        dispatch(loginActions.goToWalletSelectorStep());
+        adapters
+          .parseError(error)
+          .then((text) => {
+            dispatch(loginActions.addAccountAuthFailure(text));
+            dispatch(
```

Comment:
Why we don't call the `processError` here and in L159 of the same file and we parse the error instead and open the snackbar?

Reply:
Because `processError` would always open the error-report snackbar and here we might need to show an `info-msg` to the user. In other words, we only want to show the error-report snackbar when we have an exception or decoding error (`Error` or `ZodError`) but here we may need to show other "non-reportable" issues to the user (as we were doing before).

Reply:
Okay perfect, i like the approach!